code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -30,14 +30,17 @@
 	ammo_x_offset = 3
 
 /obj/item/gun/energy/laser/captain
-	name = "antique laser gun"
+	name = "commemorative laser gun"
 	icon_state = "caplaser"
 	item_state = "caplaser"
-	w_class = WEIGHT_CLASS_NORMAL
-	desc = "This is an antique laser gun. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The object menaces with spikes of energy. On the item is an image of Space Station 13 with the words NTSSGolden engraved. The station is exploding."
+	w_class = WEIGHT_CLASS_NORMAL  
+	desc = "This is a commemorative laser gun issued by Nanotrasen as a reward for exemplary service. It is decorated with leather and chrome. On the side is an image of Space Station 13 with the words NTSSGolden engraved. All craftsmanship is of the highest quality." // Never forget Golden, even when changing this description. 
 	force = 10
 	ammo_x_offset = 3
+	cell_type = /obj/item/stock_parts/cell{charge = 700; maxcharge = 700} //This retain's the guns original capacity of 14 shots. Also just like before it will actually feel like 15 because at least one shot will recharge when discharging a full magazine
```

Comment:
Make subtype of cell and instance object out of that instead of this.

On the other hand I had no idea dm supports that lol

Reply:
> On the other hand I had no idea dm supports that lol

This is how every energy gun with non-standard charge values does it. Take look at the code for the HoS laser, or the mini-energy gun. Doing something different here would make it different from every other similar instance in the code and be inconsistent.

Reply:
I have completely refactored how cell charge is defined in the latest commit